### PR TITLE
WIP copy changes

### PR DESCRIPTION
**Pull request by lukasschor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
-# Safe{Core} Protocol Specs
+# Safe{Core} Protocol Specification
+
+See also:
+- [Protocol Implementation](https://github.com/5afe/safe-core-protocol)
```
Comment (rmeissner) [inline README.md:4]: This will be move to safe-global, should be updated later

File: README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
 ## Motivation
 
 Safe{Core} Protocol is an open, modular framework to make smart accounts secure, portable, and composable.
 
-## Objective
+## Objectives
 
-The goal is to create a comprehensive system, designed to enhance the safety and flexibility of user interactions within the smart account ecosystem. The protocol provides a common framework for building additional tools, encourages the composition of various integrations, and supports all types of smart accounts. The protocol **enforces the correct conduct and procedures** of a system of rules of the **smart account ecosystem**.
+The goal is to create a comprehensive system, designed to enhance the safety and flexibility of user interactions within the smart account ecosystem. The protocol provides a common framework for building additional tools, encourages the composition of various integrations, and supports different types of smart accounts. In order to achieve interoperability and security, the protocol enforces a set of rules within the smart account ecosystem.
```
Comment (rmeissner) [inline README.md:13]: should we bold `interoperability` and `security`.

You also mentioned `discoverability` before ... should we include this?

File: objective/README.md [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
-        - Following specific contract interfaces
-        - Best practices for development
-            - Tests
-- What is `enforcement`?
-    - **Incentives** and **Penalties**
-        - Staking for responsibility
-            - Enforce on-chain
-        - Take responsibility
-            - Pay for incorrectness
+In order for Ethereum to succeed there needs to be a (transition)[https://vitalik.ca/general/2023/06/09/three_transitions.html] of user accounts to smart-contract based accounts (“smart accounts”). This transition is enabled through account abstraction and supported by efforts like ERC-4337, ERC-6900, and others. However, there are some challenges that have to be addressed in order to ensure the success of the smart account ecosystem.
```
Comment (rmeissner) [inline objective/README.md:3]: We should link to ERC-4337 and ERC-6900

**Review by rmeissner — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (lukasschor) [reply README.md:13]: Would not include this. At least not in connection with this point (enforcement of rules).

File: objective/README.md (hunk 3 above)
Comment (lukasschor) [reply objective/README.md]: done

File: README.md (hunk 1 above)
Comment (lukasschor) [reply README.md:4]: Will keep it in mind

**Pull request merged**